bin/next-start

```diff
@@ -18,10 +19,8 @@ const argv = parseArgs(process.argv.slice(2), {
 const dir = resolve(argv._[0] || '.')
 
 const srv = new Server({ dir })
-srv.start(argv.port)
-.then(() => {
-  console.log('> Ready on http://localhost:%d', argv.port)
-})
+
+run(srv, argv.port)
```

Comment:
`next start` is expected to be used for production, thus I think just throwing an error can be better.
Or does this work even on automated deployment too ?